index.js

```diff
@@ -407,6 +407,34 @@ module.exports = {
         if (err) return cb(clarify(err, 'Error finding or creating additions feed when starting ssb-tribes2'))
         return cb()
       })
+
+      ssb.metafeeds.findOrCreate((err, myRoot) => {
+        // prettier-ignore
+        if (err) return cb(clarify(err, 'todo'))
+
+        pull(
+          ssb.db.query(
+            where(and(isDecrypted('box2'), type('group/exclude'))),
+            live({ old: true }),
+            toPullStream()
+          ),
```

Comment:
I'm a bit worried how this will interact with listInvites (hmm that function should check if `removed: true`) and epoch discovery #78 (i guess that should also check `removed`)

I'll add todos for those.

Will this interact well if we get added back into a group though?

Reply:
andre and i chatted a bit

> I haven't looked at code, but couldn't this be discovered with epoch data?
> If you're invited to epoch N, it doesn't matter if you were excluded or not on epoch X<N

yeah this is maybe another case where we need preferredEpoch
only care about an exclude if it's in the preferredEpoch

> Yeah
> For implementing the fork-merging rules we might anyway need to do more epoch kung fu, so we need the basic epoch-related queries for that
> Not sure if I make sense

or maybe not necessarily preferredEpoch, since you can
1. get added to and removed from epoch 0
2. get an invite to epoch 1
3. epoch 1 rotates to epoch 2, you get re-added to it
4. you accept the invite to epoch 1, even though epoch 2 is preferred

Reply:
> By the time you accept invite to epoch 1, then epoch 1 is preferred at *that* time
> As soon as you discover epoch 2, then that switches
> Preferred epoch is a subjective local-peer concept, it's not global for all peers
> But by design we want them to converge to the same

Reply:
made issue for this https://github.com/ssbc/ssb-tribes2/issues/89